new_platforms/vsextension/ProjectWizard/ImportEnclaveCommand.cs

```diff
@@ -393,11 +393,20 @@ private async void Execute(object sender, EventArgs e)
                     {
                         var config3 = config as VCConfiguration3;
                         string name = config.Name;
+
+                        var clRule = config.Rules.Item("CL") as IVCRulePropertyStorage;
+                        string preprocessorDefinitions = clRule.GetUnevaluatedPropertyValue("PreprocessorDefinitions");
+                        if (isWindows)
+                        {
+                            // Allow calling getenv() in the generated host code.
+                            preprocessorDefinitions = "_CRT_SECURE_NO_WARNINGS;" + preprocessorDefinitions;
+                            clRule.SetPropertyValue("PreprocessorDefinitions", preprocessorDefinitions);
+                        }
+
                         if (name.Contains("ARM"))
                         {
-                            var clRule = config.Rules.Item("CL") as IVCRulePropertyStorage;
-                            string value = clRule.GetUnevaluatedPropertyValue("PreprocessorDefinitions");
-                            clRule.SetPropertyValue("PreprocessorDefinitions", "_ARM_;" + value);
+                            preprocessorDefinitions = "_ARM_;" + preprocessorDefinitions;
```

Comment:
I hadn't noticed this before: GCC defines `__arm__` and MSVC defines `_M_ARM`. Why is `_ARM_` necessary?

[Reference](https://sourceforge.net/p/predef/wiki/Architectures/)

Reply:
not sure, but since it's not new, I'll investigate after this change

Reply:
In the new_platforms branch, tcps.h has this:
tcps.h:#if defined(OE_SIMULATE_OPTEE) || defined(_ARM_)

This is not needed in the master branch.